### PR TITLE
Mixpanel analytics fix

### PR DESCRIPTION
**Pull request by MichaelAEden**
The falsely reported `Closed room` events all had `max_size=1`.

Since we're not interested in `Closed room` events with `max_size=1`, an easy fix would be to only report events with `max_size > 1`.

Changes have been tested locally.

Changes
- Only report `Closed room` events if `max_size > 1`
- Change event name to match Peter's convention; there is no way to [delete events in Mixpanel](https://stackoverflow.com/questions/44870497/delete-historical-events-in-mixpanel) so this is the next best thing

### CODE REVIEW

File: src/analytics.ts [hunk 1]
```diff
@@ -53,13 +53,16 @@ export const onRoomExited = (roomID: string, roomSize: number) => {
...
-    reportRoomEvent(
-      "Room closed",
-      roomMetadata.created,
-      roomID,
-      roomMetadata.maxSize
-    );
     delete roomsMetadata[roomID];
+
+    // Only report event if room had more than one user
+    if (roomMetadata.maxSize > 1)
```
Comment (MickeyDang) [inline src/analytics.ts:59]: Hm. I don't think I entirely get why does this cause a bug where a room with the same ID gets closed multiple times. If the max size is 1, and someone leaves, why would the event fire again for the same room id?

Comment (MichaelAEden) [reply src/analytics.ts:59]: I have no idea. I figured this was a much easier and quicker solution than to spend time investigating why this was occurring.

Comment (MickeyDang) [reply src/analytics.ts:59]: Resolution:
We don't know why events fire multiple times. We just know that the maxsize is always 1 and doing this fixes the problem.

**Review by MickeyDang — APPROVED** [review]
🚀  !

**Pull request merged**